<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_ajouter</class>
 <widget class="QDialog" name="login_ajouter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_ajouter">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/button_init.png);</string>
   </property>
   <property name="text">
    <string>   Valider</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_autorisation">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_etat">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>510</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_text_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Identifiant</string>
   </property>
   <widget class="QLineEdit" name="text_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>2</y>
     <width>801</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/Background_color.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_text_id_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mot de passe</string>
   </property>
   <widget class="QLineEdit" name="text_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_text_id_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Autorisation</string>
   </property>
   <widget class="QRadioButton" name="radio_admin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Administrateur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_docteur">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Docteur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_secretaire">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Secr√©taire</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_text_id_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Etat</string>
   </property>
   <widget class="QRadioButton" name="radio_actif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Actif</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_suspendu">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Suspendu</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="icon_add_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/Gestion login/icon_add.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dashboard/btn_gestion_patient1.png)3 10 15 10;
font: 18pt &quot;Bahnschrift&quot;;
</string>
   </property>
   <property name="text">
    <string>  Ajouter un Login</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>-10</y>
     <width>20</width>
     <height>651</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="icon_add_login_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/login/img/login/checked.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>text_etat</zorder>
  <zorder>text_autorisation</zorder>
  <zorder>background</zorder>
  <zorder>btn_ajouter</zorder>
  <zorder>group_text_id</zorder>
  <zorder>group_text_id_2</zorder>
  <zorder>group_text_id_3</zorder>
  <zorder>group_text_id_4</zorder>
  <zorder>btn_add_login</zorder>
  <zorder>icon_add_login</zorder>
  <zorder>line</zorder>
  <zorder>icon_add_login_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
